src/rotation_types.jl

```diff
@@ -284,9 +300,9 @@ append!(defined_conversions, [(AngleAxis, RodriguesVec), (RodriguesVec, AngleAxi
 # rotation properties
 @inline norm(rv::RodriguesVec) = sqrt(rv.sx * rv.sx + rv.sy * rv.sy + rv.sz * rv.sz) # norm is meaningful for these things
 @inline rotation_angle(rv::RodriguesVec) = norm(rv)
-function rotation_axis{T}(rv::RodriguesVec{T})     # what should this return for theta = 0?
+function rotation_axis(rv::RodriguesVec)     # what should this return for theta = 0?
     theta = rotation_angle(rv)
-    return (theta > eps(T) ? Vec(rv.sx / theta, rv.sy / theta, rv.sz / theta) : Vec(one(theta), zero(theta), zero(theta)))
+    return (theta > eps(typeof(theta)) ? Vec(rv.sx / theta, rv.sy / theta, rv.sz / theta) : Vec(one(theta), zero(theta), zero(theta)))
```

Comment:
And what about the other eps checks you have above?  Maybe you should just check for equality with zero instead - near zero you have the special case that you can represent very small numbers without cancellation error, so eps may not be appropriate.